games.json

```diff
@@ -0,0 +1 @@
+{"games":"[{\"id\":335256,\"multiplayer\":2000,\"publish_date\":2000,\"archived\":false,\"last_played_at\":2022,\"author_id\":924870,\"first_name\":\"dgdfsgrfhn\",\"last_name\":\"tntrntr\"},{\"id\":981647,\"multiplayer\":2015,\"publish_date\":2015,\"archived\":false,\"last_played_at\":2023,\"author_id\":450941,\"first_name\":\"first\",\"last_name\":\"last\"},{\"id\":827194,\"multiplayer\":2021,\"publish_date\":2021,\"archived\":false,\"last_played_at\":2023,\"author_id\":969748,\"first_name\":\"nveonve\",\"last_name\":\"woebnfwbfn\"},{\"id\":497144,\"multiplayer\":2020,\"publish_date\":2020,\"archived\":false,\"last_played_at\":2002,\"author_id\":440569,\"first_name\":\"d\",\"last_name\":\"d\"}]"}
```

Comment:
- [ ] Kindly fix the case when there's no `file.json` created and you try to run the app. It should create empty files to receive the new data. I'm doing references to books.json, games.json and music_albums.json

![image](https://github.com/lordksix/catalog-things-ruby/assets/93630700/0f55cd5c-ad19-4c58-bd2a-5503347bc2f7)

Reply:
Thank you for the feedback. The bug has been fixed.